app/controllers/admin/services_controller.rb

```diff
@@ -2,11 +2,17 @@ class Admin::ServicesController < Admin::BaseController
   before_action :set_service, only: [:show, :update]
 
   def index
-    @services = Service.order(updated_at: :desc).page(params[:page])
+    @services = Service.order(updated_at: :desc)
```

Comment:
I would maybe move the order after the search and possibly combine with the pagination. So:
```suggestion
      @services = @services.search(params[:query]) if params[:query].present?
      @services = @services.order(updateed_at: :desc).page(params[:page])
```

Reply:
oh wait that won't work as don't have @services set yet...

Reply:
Maybe:
```suggestion
    if params[:query].present?
      @services = @services.search(params[:query])
    else
      @services = Service.all?
    end
    @services = @services.order(updated_at: :desc).page(params[:page])
```
Just trying to reduce queries... although subject to change depending on filters etc